From 080e40149948c0628efff4c4c443ab5132567227 Mon Sep 17 00:00:00 2001
From: Guido Guenther <agx@sigxcpu.org>
Date: Thu, 25 Sep 2008 12:24:42 +0200
Subject: [PATCH] start network if it's not already active

Closes: #499867
---
 src/virtManager/create.py |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/src/virtManager/create.py b/src/virtManager/create.py
index 9bf6ed5..6c3448b 100644
--- a/src/virtManager/create.py
+++ b/src/virtManager/create.py
@@ -627,6 +627,12 @@ class vmmCreate(gobject.GObject):
             except ValueError, e:
                 return self.err.val_err(_("Failed to set up usermode networking"), str(e))
 
+        if self._net.type == "network":
+            if self._net.network not in self.connection.vmm.listNetworks():
+                net = self.connection.vmm.networkLookupByName(self._net.network)
+                net.create()
+                logging.info("Startet network '%s'." % self._net.network)
+
         if self._disk is not None:
             guest.disks = [self._disk]
         else:
-- 
1.5.6.5

