From: Cole Robinson <crobinso@redhat.com>
Date: Mon, 6 Feb 2012 17:57:14 -0500
Subject: vnc: Fix accidental recursion we reporting grab keys

---
 src/virtManager/console.py |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/virtManager/console.py b/src/virtManager/console.py
index c00011e..3c40c6b 100644
--- a/src/virtManager/console.py
+++ b/src/virtManager/console.py
@@ -390,12 +390,6 @@ class VNCViewer(Viewer):
     def open_fd(self, fd):
         self.display.open_fd(fd)
 
-    def get_grab_keys(self):
-        keystr = self.get_grab_keys()
-        if not keystr:
-            keystr = "Control_L+Alt_L"
-        return keystr
-
     def set_credential_username(self, cred):
         self.display.set_credential(gtkvnc.CREDENTIAL_USERNAME, cred)
 
