From: Jan ONDREJ <ondrejj@salstar.sk>
Date: Wed, 10 Feb 2010 12:43:12 -0500
Subject: [PATCH] Fix tapering issue for fully filled graph.

Closes: #572065
Origin: upstream, http://hg.fedoraproject.org/hg/virt-manager/rev/ce8158e89bc1
---
 src/virtManager/graphwidgets.py |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/virtManager/graphwidgets.py b/src/virtManager/graphwidgets.py
index 141477e..0dd4273 100644
--- a/src/virtManager/graphwidgets.py
+++ b/src/virtManager/graphwidgets.py
@@ -384,6 +384,10 @@ class Sparkline(gtk.DrawingArea):
 
             draw_line(cairo_ct, cell_area, points)
             if self.filled:
+                # XXX: Fixes a fully filled graph from having an oddly
+                #      tapered in end (bug 560913). Need to figure out
+                #      what's really going on.
+                points = [(0, cell_area.height)] + points
                 draw_fill(cairo_ct, cell_area, points, taper=True)
 
         # Stop clipping
-- 
