From 04d2ced0399ba2333e93ae98c150730b9e1574fe Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Tue, 14 Oct 2008 20:59:41 +0200
Subject: [PATCH] qemu://session can't write to the pool so use current directory

---
 src/virtManager/config.py |    7 +++----
 1 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/src/virtManager/config.py b/src/virtManager/config.py
index da8d6db..9d7d6fd 100644
--- a/src/virtManager/config.py
+++ b/src/virtManager/config.py
@@ -380,10 +380,9 @@ class vmmConfig:
     def get_default_image_dir(self, connection):
         if connection.get_type() == "Xen":
             return DEFAULT_XEN_IMAGE_DIR
-        #elif os.access(DEFAULT_VIRT_IMAGE_DIR, os.W_OK):
-        #    return DEFAULT_VIRT_IMAGE_DIR
-        #else:
-        #    return os.getcwd()
+
+        if connection.is_qemu_session():
+            return os.getcwd()
 
         # Just return the default dir since the intention is that it
         # is a managed pool and the user will be able to install to it.
-- 
1.6.0.1

