Index: virt-manager/src/Makefile.am
===================================================================
--- virt-manager.orig/src/Makefile.am	2007-11-23 15:40:32.000000000 +0100
+++ virt-manager/src/Makefile.am	2007-11-23 15:40:54.000000000 +0100
@@ -22,7 +22,7 @@
 dbus_SOURCES = $(PACKAGE).service.in
 dbus_DATA = $(PACKAGE).service
 
-schemadir = $(sysconfdir)/gconf/schemas
+schemadir = $(datadir)/gconf/schemas
 schema_SOURCES = $(PACKAGE).schemas.in
 schema_DATA = $(PACKAGE).schemas
 
